ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_facts.py

```diff
@@ -1941,34 +1941,47 @@ def _overlay_vpn_ipv6(self) -> bool:
         )
 
     @cached_property
-    def overlay(self) -> dict:
+    def overlay_peering_address(self) -> str | None:
```

Comment:
This should not be exposed in the generated facts, so it should be prefixed with underscore.

```suggestion
    def _overlay_peering_address(self) -> str | None:
```

Reply:
Fixed!